src/lang/builtins_prometheus.ml

```diff
@@ -64,19 +64,15 @@ let add_metric metric_name create register set =
       let m =
         create ~label_names ?registry:None ~help ?namespace ?subsystem name
       in
-      Lang.val_fun
-        [("label_values", "label_values", Lang.list_t Lang.string_t, None)]
-        ~ret_t:set_t
-        (fun p _ ->
+      Lang.val_fun [("label_values", "label_values", None)] (fun p ->
```

Comment:
Let's make prometheus a `on_frame` scripted hook if we can after this.

Reply:
Yeah!